test/typescript/multiple-stacks/test.ts

```diff
@@ -5,6 +5,9 @@
 //
 import { TestDriver } from "../../test-helper";
 
+const onWindows = process.platform === 'win32' ? it.skip : it.skip
```

Comment:
Seems incorrect for both branches to be the same here.

Reply:
Oh, you're right. Didn't catch that when stealing it from https://github.com/hashicorp/terraform-cdk/blob/5bb0d6877ff334869ec555af53cf885616ba1a7f/test/typescript/modules/test.ts#L12-L12

Reply:
Added a separate PR to fix the test I stole it from: https://github.com/hashicorp/terraform-cdk/pull/740